backends/cuda-gen/ceed-cuda-gen-operator-build.cpp

```diff
@@ -887,9 +854,11 @@ extern "C" int CeedCudaGenOperatorBuild(CeedOperator op) {
   code << "data.tidz = threadIdx.z;\n";
   code << "data.tid  = threadIdx.x + threadIdx.y*blockDim.x + threadIdx.z*blockDim.y*blockDim.x;\n";
   code << "data.slice = slice+data.tidz*Q1d"<<(dim>1?"*Q1d":"")<<";\n";
+
+  code << "\n// Input field constants and basis data\n";
```

Comment:
@YohannDudouit, hopefully you don't mind. I added a bunch of these types of comments to the generated code to help me debug. I think it will help us with #460.